Makefile

```diff
@@ -72,10 +72,7 @@ sgx-cli: sgx-env
 	$(MAKE) -C test-collateral/generate-policy
 
 nitro: sdk
-	pwd
 	RUSTFLAGS=$(NITRO_RUST_FLAG) $(MAKE) -C runtime-manager nitro
```

Comment:
Is building `runtime-manager` better as a dependency on the `nitro` build target (similar to `sdk`)?

Reply:
I'm not sure what you mean. Can you clarify?

Reply:
Sorry, ignore that.  I misread the line I was commenting on...